pvnet/models/utils.py

```diff
@@ -110,7 +118,7 @@ def flush(self) -> dict[BatchKey, list[torch.Tensor]]:
         """Concatenate all accumulated batches, return, and clear self"""
         batch = {}
         for k, v in self._batches.items():
-            if k == BatchKey.gsp_t0_idx:
+            if k == BatchKey.gsp_t0_idx or k == BatchKey.wind_t0_idx or k == BatchKey.pv_t0_idx:
```

Comment:
can we use `if k in [BatchKey.gsp_t0_idx, BatchKey.wind_t0_idx, BatchKey.pv_t0_idx]` to be a little cleaner

Reply:
Or even `k==f"{self.key_to_keep}_t0_idx"`